### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by John-Kibirige**
## Main tasks carried out
- Add forms in the required semantic way, that is, adding labels and using the form element instead of a div element.
- Carry out inbuilt client-side form validation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -228,14 +228,31 @@ <h1 class="contact-title">
...
+                     <input type="text" placeholder="Text Input" class="user-name" id="user-name" name="user-name" maxlength="30" required >
+                  </li>
+                  <li>
+                     <label for="email"></label>
+                     <input type="email" name="email" id="email" placeholder="kibirigejohn005@gmail.com" class="user-email" required >
+
+                  </li>
+                  <li>
+                     <label for="user-text"></label>
+                     <textarea name="text-area" id="user-text" cols="30" rows="10" class="user-text-area" placeholder="Enter text here" maxlength="500" required></textarea>
```
Comment (vechicin) [inline index.html:244]: Please provide appropriate placeholders (e.g. `Enter your name here`) to improve UX. 

Screen Shot 2022-07-05 at 13 31 00

File: index.html [hunk 2]
```diff
@@ -228,14 +228,31 @@ <h1 class="contact-title">
...
 
-            <div class="contact-form">
-               <input type="text" placeholder="Text Input" class="user-text">
-               <input type="text" name="email" id="email" placeholder="kibirigejohn005@gmail.com" class="user-email">
-               <textarea name="text-area" id="user-text" cols="30" rows="10" class="user-text-area" placeholder="Enter text here"></textarea>
-               <button class="see-projects project-btn">
-                  See Project
-               </button>
-            </div>
+            <form class="contact-form" action="https://formspree.io/f/xnqwkjoy" method="post">
```
Comment (vechicin) [inline index.html:231]: This is how your design looks on my end (using Google Chrome).

Screen Shot 2022-07-05 at 13 35 24

Please fix it.

Screen Shot 2022-07-05 at 13 35 43

File: index.html [hunk 3]
```diff
@@ -228,14 +228,31 @@ <h1 class="contact-title">
...
+                  </li>
+                  <li>
+                     <label for="user-text"></label>
+                     <textarea name="text-area" id="user-text" cols="30" rows="10" class="user-text-area" placeholder="Enter text here" maxlength="500" required></textarea>
+
+                  </li>
+                  <li>
+                     <button class="see-projects project-btn" type="submit" value="send">
+                        See Project
+                     </button>
```
Comment (vechicin) [inline index.html:250]: Your button is missing the `active` interaction styling. Please add it.

Screen Shot 2022-07-05 at 13 39 45

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @John-Kibirige ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (John-Kibirige) [reply index.html:231]: Hey @vechicin  !! Thanks for the comment, however, I hope you adjusted the screen size to 375px as the ideal mobile size.

File: index.html [hunk 4]
```diff
@@ -228,14 +228,31 @@ <h1 class="contact-title">
...
+
+                  </li>
+               </ul>
+             
+            </form>
```
Comment (mirouhml) [inline index.html:255]: - [ ] Please know that as the previous reviewer mentioned, your website should be responsive for almost all mobile phones, which means the minimum width should be `375px` but that doesn't mean it's the only width, your design should be responsive to up to the width `768px`, in your case to solve this you should remove the `fixed width` you put in your CSS, and use flexbox to align the content in the webpage accordingly this way any screens between the widths I mentioned would display your website correctly. 😁 Also, it's worth knowing that later you'll have to adapt your webpage to be responsive for desktop as well (you don't have to worry about that now) this means if you set up the correct foundation now by using the correct methods, it'll be easy for you to adapt your website to any screen size. 😉

- [ ] Also you forgot to update the placeholder for the email input, as it should be something like `Email`, kindly change that. 😁

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @John-Kibirige, 

Great work on making the changes requested by the previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet, but don't worry you're almost there!! 😉

image

 ## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (John-Kibirige) [reply index.html:255]: Thank you so much  @mirouhml for this detailed review

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @John-Kibirige,

Good job following the changes, your website looks so much better now and it's responsive as well! 

Your project now is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] **Please make sure next time in group projects to have commits from your coding partners as well in your project, as it is required to do so.**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**